CHANGELOG.md

```diff
@@ -25,6 +25,12 @@ The types of changes are:
 
 - Merge instances of RTK `createApi` into one instance for better cache invalidation [#3059](https://github.com/ethyca/fides/pull/3059)
 
+## [2.12.1](https://github.com/ethyca/fides/compare/2.12.0...2.12.1)
```

Comment:
heads up on the changelog here -- i _think_ this should be right, but let's confirm this is how we want to do this before we merge

Reply:
Do we need to adjust the unreleased line too then?

Reply:
i decided to just put this under `Unreleased` and we can make the changelog updates on the the hotfix branch. i'm not sure what i was thinking originally, trying to be clever but it felt misguided on a second look. thanks for the 👀 !